monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -736,5 +736,76 @@
         },
       ],
     },
+    {
+      name: 'rbdmirror',
+      rules: [
+        {
+          alert: 'CephRBDMirrorConnectionLost',
+          'for': '1m',
+          expr: 'mirror_connection_state != 1',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: 'RBD Mirror connection is lost, but mirror daemon is up',
+            description: "RBD Mirror connection is lost, but mirror daemon is up",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesPerDaemonHigh',
+          'for': '1m',
+          expr: 'count(rbd_mirror_image_status {daemon=xx) >= 100| |high image lag time|possible resource constraints or high data transfers|(rbd_mirror_image_status\{image=X, pool=Y, last_sync_duration)} == 1) > rbd_mirror_image_schedule{image=X, pool=Y, interval}',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.1.2' },
+          annotations: {
+            documentation: '#TODO',
+            summary: '#TODO',
+            description: "#TODO",
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSync',
```

Comment:
the image status will have these values, ```0 = Unknown
1 = Error
2 = Syncing
3 = Starting Replay
4 = Replaying
5 = Stopping Replay
6 = Stopped
```
so i think, instead of not in sync alert. we should raise an alert when the status is in error. we can have `CephRBDMirrorImagesNotInSync` for the images, whose local and remote timestamp differ. Those images are not yet fully synced, probably are in the processes of syncing.

Reply:
@weirdwiz , @jmolmo , can you review the current expression/query and see whether we require any improvement?
Thanks